docs/README

```diff
@@ -6,7 +6,7 @@ This allows it to be built into other forms for easier viewing and browsing.
 
 To create an HTML version of the docs:
 
-* Install Sphinx (using ``python -m pip install Sphinx`` or some other method).
+* Install Sphinx (using ``python -m pip install sphinx`` or some other method).
```

Comment:
Why was `Sphinx` lowercased here? I recognize that either will work, but the package entry on PyPI is capitalized. When referencing other projects, I think we should defer to a project's own preferred spelling. See https://pypi.org/project/Sphinx/ and notice that it is capitalized in both the URL and on the page, something not all entries do. This comment applies to several other lines in the change.